### DISCUSSION TITLE
Legal text-like list and lettered list

### DISCUSSION THREAD

**Discussion by matjaeck** [remarkjs/remark#878] [Ideas] [answered]
Hi,

I want to make some legal texts online available and for that reason I would like to parse both of the following formats into ordered lists:

```

(1) Here comes the text for list item number one...

(2) And there is more...

  a) text for a lettered list item 1

  b) and so on
```

Could someone give me a pointer on where to look for similar plugins or how to accomplish this?

Thanks!

Edit: I should note that the lettered-list-parser has higher priority for me, and is probably also more useful for other people.

**Comment by wooorm** [comment, marked as answer]
Short answer: Don’t.
Long answer: See the (rather long) article on how to extend micromark and whether you should: https://github.com/micromark/micromark#extensions.
